community/modules/scripts/ramble-setup/templates/ramble_setup.yml.tpl

```diff
@@ -36,32 +22,38 @@
     chown_owner: ${chown_owner}
     chgrp_group: ${chgrp_group}
   tasks:
+  - name: Acquire lock
+    ansible.builtin.shell:
+      mkdir -p "{{ install_dir | dirname }}/.ramble_lock"
```

Comment:
Install_dir is already a dir. Have you tested behavior of dirname if passed a dir. My worry is that if I set `/apps/my/ramble` it will remove the `ramble` where as mkdir would tolerate this.

Reply:
Per our discussion, I left the dirname filter.